src/interface/src/app/plan/create-scenarios/constraints-panel/constraints-panel.component.html

```diff
@@ -44,17 +43,26 @@
           <div>
             <mat-form-field class="input-field" appearance="outline" floatLabel="always">
               <mat-label>Distance from roads </mat-label>
-              <input class="right-align" formControlName="maxRoadDistance" matInput type="number">
+              <input class="right-align" formControlName="minDistanceFromRoad" matInput type="number">
               <span matSuffix> yds</span>
             </mat-form-field>
           </div>
         </div>
-        <div class="button-wrapper">
-          <div class="small-label">Stand Size</div>
-          <mat-button-toggle-group formControlName="standSize" name="standSize" aria-label="Stand Size">
-            <mat-button-toggle *ngFor="let item of standSizeOptions;" [value]="item"
-              checked>{{item}}</mat-button-toggle>
-          </mat-button-toggle-group>
+        <div class="flex-row">
+          <div>
+            <mat-form-field class="input-field" appearance="outline" floatLabel="always">
+              <mat-label>Max acres to be treated </mat-label>
+              <input class="right-align" formControlName="maxArea" matInput type="number">
+            </mat-form-field>
+          </div>
+          <mat-form-field class="input-field" appearance="outline">
+            <mat-label>Stand size</mat-label>
+            <mat-select formControlName="standSize" [(value)]="standSizeOptions[-1]" required>
```

Comment:
Not sure if I understand this : `standSizeOptions[-1]`. Should it be [0] ?

Reply:
Realized the default value doesn't actually have to get set here since it already gets set in the form builder — removed!